src/core/store/modules/lines/lines.actions.ts

```diff
@@ -60,8 +60,12 @@ const setSelectedLinePositionProposal = createAction<{ position?: string; propos
 );
 const setPosition = createAction<{ id: string; position: CreditPosition }>('lines/setPosition');
 
+const makeProposal = createAction<{ maker: string; position: AddCreditProps }>('lines/makeProposal');
```

Comment:
How do we not already have an `addCredit` action? Should be reusing that

Reply:
deleted `makeProposal` and moved into `addCredit` action/reducer